xrayvision/Visibillity.py

```diff
@@ -57,6 +58,23 @@ def to_map(self, outmap):
         """
         return Transform.idft(outmap, self.uv, self.vis)
 
+    def to_sunpy_map(self, outmap: sunpy.map):
+        """
+        Converts the visibility data into an actual image (specified by
+        the given outmap sunpy map
+
+        Parameters
+        ----------
+        outmap
+
+        Returns
+        -------
+
+        """
+        data = Transform.idft(outmap.data, self.uv, self.vis)
```

Comment:
The map coordinates need to be taken into account:
* map centre coordinate
* pixel size